templates/test/module_table.html.ep

```diff
@@ -22,6 +22,9 @@
                         %= link_to $module->{name} => url_for('src_step', stepid => 1, moduleid => $module->{name}, testid => $testid)
                     </div>
                     <div class="flags">
+                        % if ($module->{spent_time}) {
+                            <span>spent time: <%= $module->{spent_time} %><span>&nbsp;
```

Comment:
The 2nd `<span>` should likely be `</span>`.

Additionally, I'd rather call it `Execution time: ` (and would also rename the internal variable to `execution_time`). But better wait for more feedback regarding that before renaming it because I'm not quite sure about the best wording.